opinions/spring_boot_opinions.go

```diff
@@ -81,8 +81,7 @@ var SpringBoot = Opinions{
 			if c.LivenessProbe.Handler == (corev1.Handler{}) {
 				c.LivenessProbe.Handler = corev1.Handler{
 					HTTPGet: &corev1.HTTPGetAction{
-						// TODO(scothis) is this the best path for liveness?
-						Path: "/actuator/info",
+						Path: "/manage/info",
```

Comment:
I don't believe this will work in the default case. The default path is `/actuator/info`, it is petclinic that is overriding the default https://github.com/spring-projects/spring-petclinic/blob/master/src/main/resources/application.properties#L16.

I think we need to either override `management.endpoints.web.base-path` or find a way to read the existing values. This information is not currently surfaced in the metadata.

Reply:
updated to override the value, and added a todo to not clobber in the future